src/ray/object_manager/object_manager.cc

```diff
@@ -453,17 +454,7 @@ void ObjectManager::ConnectClient(std::shared_ptr<TcpClientConnection> &conn,
 
 void ObjectManager::DisconnectClient(std::shared_ptr<TcpClientConnection> &conn,
```

Comment:
Should we just get rid of this method and just call `connection_pool_.RemoveReceiver(conn)` directly?

Reply:
Hmm I usually prefer to not have methods that are both only called once and also just call another method. I suppose it's okay though, since we'll probably have to reuse the method for cleaning up failed connections later on.